Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1532,4 +1536,98 @@ namespace Babylon
     {
         return Napi::Value::From(info.Env(), m_graphicsImpl.GetHardwareScalingLevel());
     }
+
+    Napi::Value NativeEngine::CreateImageBitmap(const Napi::CallbackInfo& info)
+    {
+        auto env = info.Env();
+        if (!info[0].IsArray())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap first parameter must be an array.");
+        }
+        const auto parameters = info[0].As<Napi::Array>();
+        if (!parameters.Length())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap parameters are empty.");
+        }
+
+        const auto blob = parameters[0u].As<Napi::Array>();
+        if (!blob.Length())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap blob parameter is empty.");
+        }
+
+        const auto data = blob[0u].As<Napi::ArrayBuffer>();
+        if (!data.ByteLength())
+        {
+            throw Napi::Error::New(env, "CreateImageBitmap array buffer is empty.");
+        }
+
+        bimg::ImageContainer* image = bimg::imageParse(&m_allocator, data.Data(), static_cast<uint32_t>(data.ByteLength()));
```

Comment:
What is the image format in this case?

Reply:
I'm not sure to understand. image here will contain decoded data from coded data